tests/supply_chain/test_supply_chain.py

```diff
@@ -1217,19 +1238,17 @@ def test_distribution_unit_dispatch_order(self):
 
     def test_seller_unit_initial_states(self):
         env = build_env("case_02", 100)
+        assert isinstance(env.business_engine, SupplyChainBusinessEngine)
 
         # find seller for sku3 from retailer facility
-        sell_unit: SellerUnit
-        source_facility: FacilityBase
+        sell_unit: Optional[SellerUnit] = None
 
-        for id, info in env.summary["node_mapping"]["facilities"].items():
+        for _, info in env.summary["node_mapping"]["facilities"].items():
             if info["name"] == "Retailer_001":
                 for pid, pdetail in info["units"]["products"].items():
                     if pdetail["sku_id"] == SKU3_ID:
-                        sell_unit = env._business_engine.world.get_entity(pdetail["seller"]["id"])
-
-            if info["name"] == "Warehouse_001":
```

Comment:
remove all "Warehouse_001" away?

Reply:
Sorry I did not fully get your point. Why should we do that?

Reply:
There are 2 places (around line 1231 and line 1340), you remove the part for "Warehouse_001", I want to know why do did this?

Reply:
Because "Warehouse_001" is used to get the `source_facility` in these two places, but they are never used in the unit test, so I remove the unused statements.